fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Alias.java

```diff
@@ -49,7 +49,7 @@ public Alias(Expression child, String name) {
     }
 
     @VisibleForTesting
-    Alias(ExprId exprId, Expression child, String name) {
+    public Alias(ExprId exprId, Expression child, String name) {
```

Comment:
unnecessary change

Reply:
We should make this function `public` here, due to the package of `org.apache.doris.nereids.parser.HavingCluaseTest` is different from the one of `org.apache.doris.nereids.trees.expressions.Alias`